cmd/url.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/url"
+	"github.com/spf13/cobra"
+)
+
+var (
+	urlListComponent   string
+	urlListApplication string
+)
+
+var urlCmd = &cobra.Command{
+	Use:   "url",
+	Short: "expose component to the outside world",
+	Long:  "expose component to the outside world",
+}
+
+var urlCreateCmd = &cobra.Command{
+	Use:   "create <component name, optional>",
```

Comment:
if it is optional it should be `[component name]`
`[component name]`  means optional
`<component name>` means required

Reply:
fixed